### PR TITLE
Possibly cleaned some stuff

### PR DESCRIPTION
**Pull request by xbony2**

### CODE REVIEW

File: src/Java/gtPlusPlus/core/util/Utils.java [hunk 1]
```diff
@@ -472,17 +472,11 @@ public static Integer appenedHexNotationToInteger(final int hexAsStringOrInt){
...
-		return false;
+		return OreDictionary.getOres(OreDictName).size() != 0;
 	}
 
 	public static boolean invertBoolean(final boolean booleans){
-		if (booleans == true){
-			return false;
-		}
-		return true;
+		return !booleans
```
Comment (xbony2) [inline src/Java/gtPlusPlus/core/util/Utils.java]: I don't really seem the point of this method in generally but hey

Comment (draknyte1) [inline src/Java/gtPlusPlus/core/util/Utils.java]: Some legacy code.

File: src/Java/gtPlusPlus/core/util/Utils.java [hunk 2]
```diff
@@ -472,17 +472,11 @@ public static Integer appenedHexNotationToInteger(final int hexAsStringOrInt){
 	}
 
 	public static boolean doesEntryExistAlreadyInOreDictionary(final String OreDictName){
-		if (OreDictionary.getOres(OreDictName).size() != 0) {
-			return true;
-		}
-		return false;
+		return OreDictionary.getOres(OreDictName).size() != 0;
```
Comment (Chocohead) [inline src/Java/gtPlusPlus/core/util/Utils.java]: Not `!OreDictionary.getOres(OreDictName).isEmpty()`? ;)

Comment (xbony2) [inline src/Java/gtPlusPlus/core/util/Utils.java]: That would be better, yeah.

**Comment by draknyte1** [comment]
None of these actually improve functionality though..
It's just cleanup of unused stuff for the most part :P

**Comment by xbony2** [comment]
https://ruben.verborgh.org/blog/2013/02/21/programming-is-an-art/

**Pull request closed without merging**